routes/index.js

```diff
@@ -563,7 +572,7 @@ router.put('/discover', function(req, res, next) {
     //apply 'show songs in' filter
     var loadMore = function(songs2d, done) {
         if (langShown !== 'all') {
-            songs2d = songs2d.filter((s) => s.lang === langShown)
+            songs2d = songs2d.filter((s) => s.lang == langShown)
```

Comment:
Okay. Technically this is ok. Why is the ObjectID being turned into a string when it's stored in langShown? Is it possible to consistently keep it one or the other type? If not, this is still acceptable.